resources/vue/sidebar.vue

```diff
@@ -435,6 +435,7 @@ module.exports = {
 
       // Create instances for all elements without already existing
       // tippy-instances.
+      // eslint-disable-next-line no-undef
```

Comment:
Better: Try to use `window.tippy` — and if that works it's fine (I'm using this in other parts of the renderer as well and it works just fine).

If it doesn't work fine, recommended is to put it as a global in the first line (I've done so where I used jQuery, it's basically the first line consisting of a comment with `/* global $ jQuery define etc...*/`)

Reply:
window.tippy worked!